data/v2/wizards-of-the-coast/srd-2014/Rule.json

```diff
@@ -1933,7 +1933,7 @@
 },
 {
   "model": "api_v2.rule",
-  "pk": "srd_mounts-and-vehicles_tack-harness-drawn-vehicles",
+  "pk": "srd_mounts-and-vehicles_tack-harness-land-vehicles",
```

Comment:
I generally renamed "drawn" vehicles to "land" vehicles in our pks, to match the skill proficiency they use